### ISSUE TITLE
Microphone simulator not accessible

### ISSUE THREAD

**Issue by jamesadevine** [microsoft/jacdac#529]
**Describe the bug**
User feedback:

> Microphone. I pressed the start microphone button, and see a slider. These slider values however are not automatically announced like the slider values corresponding to the light sensors. This perhaps could be due to the fact that the slider may not have been marked up with Aria. There is definitely a possibility that the slider is in deed marked up with aria attributes in which case the issue may be something else. The slider values are accessible though; I have to move focus away from, and move back on to the slider to hear the values.

**Comment by pelikhan** [comment]
Fixes at https://github.com/microsoft/jacdac-docs/commit/0328b4be1ef67cbbacdbc2e84321c463f8384cc1
Please test.
